frontend/src/static/js/main.js

```diff
@@ -175,6 +175,10 @@ window.app = new window.Vue({
     generateKey(dataObj) {
       return JSON.stringify(dataObj);
     },
+
+    getRepoSenseLink() {
+      return `https://github.com/reposense/RepoSense/commits/${window.app.repoSenseVersion}`;
```

Comment:
instead of commits, can we point to `tree`?

Reply:
I have edited it, but pass by hash feature will break this link.